scripts/SimpleHistory.lua

```diff
@@ -384,6 +384,8 @@ end
 function get_file()
 	local path = mp.get_property('path')
 	if not path then return end
+	-- ignoring protocols that do not support playback recovery
```

Comment:
Is this only to ignore these protocols from saving to history?
Or these protocols cause a problem if it was added to history?